data_quality/utilities.py

```diff
@@ -31,10 +31,36 @@ def resolve_relative_path(config_filepath, config_path):
         return os.path.join(os.path.dirname(config_filepath), config_path)
 
 def load_json_config(config_filepath):
-    """Loads the json config into a dictionary"""
+    """Loads the json config into a dictionary, overwriting the defaults"""
 
+    default_config = {
+        'data_dir': '',
```

Comment:
Currently the generate command does not also generate the  `data` folder but assumes the user created it i and writes `sources.csv` and `publishers.csv` directly in folder where the command is executed. Maybe a better default would be to create a `~/data-quality-cli/data` folder and write stuff in there?